reference/programming-model.md

```diff
@@ -571,13 +613,15 @@ console.log(`Active: ${data.active}`);
 
 ```go
 // JSON parsing helpers for config are not built-in for Go currently.
-// 
+//
 // See https://github.com/pulumi/pulumi/issues/1614.
 ```
 
+The `Config` object also provides functions to get the value from configuration and mark it as a secret.  See [config.getSecret] or [config.requireSecret].
```

Comment:
Link these?  Or provide a short example?

Noting that they return an `Output<T>` instead of a `T` may also help get ahead of that potential surprise.